package.json

```diff
@@ -22,17 +23,14 @@
   },
   "dependencies": {
     "cli-table": "<=1.0.0",
-    "coffee-register": "1.0.0",
-    "coffee-script": "1.12.7",
-    "coffeescript": "1.12.7",
```

Comment:
At work, we inject a coffeescript that adds rudimentary rbac for chatops. So, removing coffeescript is sad for me as it means extra work to replace that script (maybe compile to js?). But dropping it makes sense here since this repo doesn't make use of it.

Reply:
Good to know, I just dropped it because it seemed to be a requirement of the old version of `hubot-slack`, now the new version doesn't require it so I removed it.